monai/metrics/regression.py

```diff
@@ -224,3 +225,64 @@ def compute_mean_error_metrics(y_pred: torch.Tensor, y: torch.Tensor, func) -> t
     # reduction of batch handled inside __call__() using do_metric_reduction() in respective calling class
     flt = partial(torch.flatten, start_dim=1)
     return torch.mean(flt(func(y - y_pred)), dim=-1, keepdim=True)
+
+
+class SSIMMetric(RegressionMetric):
```

Comment:
Please add the new metrics to: https://github.com/Project-MONAI/MONAI/blob/dev/monai/metrics/__init__.py

Thanks.